### PR TITLE
feat: yearn underlying assets

### PR DESCRIPTION
**Pull request by theoboldfrazier**
Adds asset support for all ironbank tokens, zapper protocol tokens, and yearn vault underlying asset tokens.

Increased from 5396 supported assets to 5919 supported assets.

### CODE REVIEW

File: packages/asset-service/src/generateAssetData/ethTokens/yearnSdk.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { Yearn } from '@yfi/sdk'
+
+// YearnMarketCapService deps
+const provider = new JsonRpcProvider(process.env.REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL)
+export const yearnSdk = new Yearn(1, { provider })
```
Comment (0xdef1cafe) [inline packages/asset-service/src/generateAssetData/ethTokens/yearnSdk.ts:6]: can you comment or pull this magic number out to a const or something? i'm assuming this is mainnet at a first guess.

Comment (theoboldfrazier) [reply packages/asset-service/src/generateAssetData/ethTokens/yearnSdk.ts]: Sure, done.

File: packages/asset-service/sample.env [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL=https://dev-api.ethereum.shapeshift.com
```
Comment (0xdef1cafe) [inline packages/asset-service/sample.env:1]: can you add a line or two of docs to this asset service readme to copy this to `.env` before running generate?

**Review by 0xdef1cafe — COMMENTED** [review]
two tiny comments/docs to add but otherwise rad

File: packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts [hunk 3]
```diff
@@ -71,3 +40,90 @@ export const extendErc20 = async (): Promise<TokenAsset[]> => {
...
+      })
+    }
+  })
+}
+
+export const getZapperTokens = async (): Promise<TokenAsset[]> => {
+  const zapperTokens: Token[] = await yearnSdk.tokens.supported()
+  return zapperTokens.map((token: Token) => {
+    return {
+      color: '#FFFFFF',
```
Comment (0xdef1cafe) [inline packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts:77]: nit pick - we can uses zappers brand color here

File: packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts [hunk 4]
```diff
@@ -71,3 +40,90 @@ export const extendErc20 = async (): Promise<TokenAsset[]> => {
     }
   })
 }
+
+export const getIronBankTokens = async (): Promise<TokenAsset[]> => {
+  const ironBankTokens: Token[] = await yearnSdk.ironBank.tokens()
+  return ironBankTokens.map((token: Token) => {
+    return {
+      color: '#FFFFFF',
```
Comment (0xdef1cafe) [inline packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts:48]: maybe yearn's blue here too

File: packages/asset-service/sample.env (hunk 2 above)
Comment (theoboldfrazier) [reply packages/asset-service/sample.env:1]: Good call, done.

File: packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts (hunk 3 above)
Comment (theoboldfrazier) [reply packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts]: I'm not sure if these colors are used anywhere (yet). But either way, done.

File: packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts (hunk 4 above)
Comment (theoboldfrazier) [reply packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts]: Done.

**Review by DaoDev44 — APPROVED** [review]

**Review by asamere — APPROVED** [review]

**Pull request merged**